### PR TITLE
Add conda badges

### PR DESCRIPTION
**Pull request by epruesse**
see #1000

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1680,6 +1680,71 @@ cache(function(data, match, sendBadge, request) {
...
+cache(function(data, match, sendBadge, request) {
+  var mode = match[1];
+  var channel = match[2];
+  var pkgname = match[3];
+  var format = match[4];
+  var url = 'https://api.anaconda.org/package/' + channel + '/' + pkgname;
+  var modes = {
+    // downloads - 'd'
+    'd': function(data, badgeData) {
+      var downloads = 0;
```
Comment (Daniel15) [inline server.js]:
```
const downloads = data.files.reduce((total, file) => total + file.ndownloads, 0)
```

File: server.js [hunk 2]
```diff
@@ -1680,6 +1680,71 @@ cache(function(data, match, sendBadge, request) {
...
+      });
+      badgeData.text[0] = data.label || "downloads";
+      badgeData.text[1] = metric(downloads);
+      badgeData.colorscheme = downloadCountColor(downloads);
+    },
+    // latest version 'v'
+    'v': function(data, badgeData) {
+      var version = data.latest_version;
+      var color = versionColor(version).color;
+      badgeData.text[1] = "v"+version;
```
Comment (Daniel15) [inline server.js]: Nit: use single quotes (`'` instead of `"`) throughout this code block

Comment (epruesse) [inline server.js]: done

File: server.js (hunk 1 above)
Comment (epruesse) [inline server.js]: done

**Comment by epruesse** [comment]
@Daniel15 Thanks for the review. I've added the requested changes.

**Comment by epruesse** [comment]
@Daniel15 Is there anything I can/should/need to do to get this merged?

**Comment by jakirkham** [comment]
Friendly nudge. 😉

**Comment by bgruening** [comment]
@Daniel15 this would be great to have for our community as well! Thanks!

**Comment by epruesse** [comment]
@Daniel15 @espadrine I'd be happy to implement anything you are missing in this PR to allow merging! 

Thank you for providing this great service to all of us!

One question on the side: Two very popular conda channels (=distributions in conda lingo), [conda-forge](https://github.com/conda-forge) and [bioconda](https://github.com/bioconda), have logos that would be useful to display. Both projects are awesome contributions to the fields of sciences depending on the tools they make available. Every bit of visibility and attractiveness to software authors they gain is worth it. If it's in line with your preferences, I'd add the logos matching the way it's done for CircleCI.

**Comment by Daniel15** [comment]
Sorry for the delay!! This looks good, so I've merged it. We just need to wait on @espadrine to deploy the site now.

**Comment by epruesse** [comment]
Great! Thanks!

**Pull request merged**